src/utils.ts

```diff
@@ -25,11 +25,8 @@ export function cmdExists(cmd: string) {
   return which.sync(cmd, { nothrow: true }) !== null
 }
 
-export function getVoltaPrefix(): string {
-  // https://blog.volta.sh/2020/11/25/command-spotlight-volta-run/
-  const VOLTA_PREFIX = 'volta run'
-  const hasVoltaCommand = cmdExists('volta')
-  return hasVoltaCommand ? VOLTA_PREFIX : ''
+export function isVoltaExists(): boolean {
```

Comment:
`doesVoltaExist` might be a bit more grammatically correct name

Reply:
I don't think identifiers are valid English sentences, which grammar doesn't always apply (it can be read as an abbr of "is it true that volta exists in current directory?"). I tend to use `is` prefix to describe functions that return boolean.

Reply:
fair enough. might be able to make it `isVoltaInstalled` or `isVoltaPresent` or something like that if you wanted to keep the `is` prefix and make it a bit more grammatical

anyway, this PR looks good to me. this was the only thing I noticed